src/ducks/apps/components/Discover.jsx

```diff
@@ -55,6 +56,7 @@ export class Discover extends Component {
                 onAppClick={this.onAppClick}
                 intentData={intentData}
                 parent="discover"
+                alternativeStore={alternativeStore}
```

Comment:
Why for `Discover` you concat apps and alternativeApps and here you share alternativeStore directly?